.jenkinsci/utils/doxygen.groovy

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env groovy
 
 def doDoxygen() {
+  sh "doxygen Doxyfile"
   // TODO: Remove this comment once dev branch will return to develop
   // I will not be changing branches here. It requires some rewriting
   // Hope dev branch situation will be resolved soon
   if (env.GIT_LOCAL_BRANCH in ["master","develop"] || env.CHANGE_BRANCH_LOCAL == 'develop') {
```

Comment:
Should be refactored to work with dev branch. But, again, I would recommend to move these checks to Jenkinsfile